marklogic-data-hub-central/ui/src/components/raw-data/raw-data-card-view/raw-data-card-view.tsx

```diff
@@ -67,6 +65,44 @@ const RawDataCardView = (props) => {
     }
   }
 
+  const displayDocumentSources = (item) => {
+    let sources = item.hubMetadata?.sources.map((el) => {
+      return el.name;
+    }).join(", ");
+    return sources;
+  }
+
+  const emptyField = (
+    <p className={styles.textDisabled}>none</p>
+  )
+
+  const displayDocumentMetadata = (item) => {
+    return (
+      <div className={styles.popover} data-testid={item.uri + '-popover'}>
+        <div className={styles.colKey}>
+          <p>Source:</p>
+          <p>Flow:</p>
+          <p>Step:</p>
+          <p>Created On:</p>
+        </div>
+        <div className={styles.colValue}>
+          {item.hubMetadata?.sources ? <span className={styles.valText} data-testid={item.uri + '-sources'}>
+            <OverflowTooltip title={displayDocumentSources(item)} placement={"bottom"} content={displayDocumentSources(item).substring(0, 28)} width={"200px"} />
```

Comment:
Likewise, "RecordSources" would be more appropriate here. 

In fact, I'd rather this component be called "record-view" instead of "raw-data-card". This data is not necessarily "raw", which we don't have a clear definition of either. It may be highly structured data, just not an entity instance.

Reply:
Changed to RecordSources and renamed component. Thanks.